lib/src/widgets/detailCardsWidgets/custom_appbar.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import '../../utils/ui_constants.dart';
+
+class CustomAppBar extends StatelessWidget {
+  const CustomAppBar({
+    Key? key,
+    this.movieImage,
+  }) : super(key: key);
+
+  final String? movieImage;
+
+  @override
+  Widget build(BuildContext context) {
+    return SliverAppBar(
+      expandedHeight: UiConstants.movieDetailsAppBarExpandedHeight,
+      flexibleSpace: FlexibleSpaceBar(
+        background: Image.network(
+          movieImage!,
```

Comment:
you can remove the "!" operator, after making the parameter non nulable

Reply:
removed.